doc/crds/ip-reconciler-job.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app: whereabouts
 spec:
   concurrencyPolicy: Forbid
-  successfulJobsHistoryLimit: 0
+  successfulJobsHistoryLimit: 3
```

Comment:
If you're now logging to the host's file-system, why would we want to preserve these around ?